core/block/export/export.go

```diff
@@ -372,6 +373,10 @@ func (e *export) getNested(spaceID string, id string, docs map[string]*types.Str
 	return nestedDocsIds
 }
 
+func (e *export) isLinkedObjectExist(rec []database.Record) bool {
```

Comment:
We can make isLinkedObjectExist package-level func, as no fields or methods of export are used